okhttp-backend/monix/src/test/scala/sttp/client/okhttp/monix/OkHttpMonixWebSocketTest.scala

```diff
@@ -0,0 +1,54 @@
+package sttp.client.okhttp.monix
+
+import java.nio.channels.ClosedChannelException
+
+import monix.eval.Task
+import monix.execution.Scheduler.Implicits.global
+import monix.reactive.Observable
+import sttp.client._
+import sttp.client.impl.monix.{MonixStreams, TaskMonadAsyncError, convertMonixTaskToFuture}
+import sttp.client.monad.MonadError
+import sttp.client.monad.syntax._
+import sttp.client.okhttp.OkHttpBackend
+import sttp.client.testing.{ConvertToFuture, WebSocketStreamingTest}
+import sttp.client.testing.HttpTest.wsEndpoint
+import sttp.client.testing.websocket.WebSocketTest
+import sttp.client.ws.WebSocket
+import sttp.model.ws.WebSocketFrame
+
+import scala.concurrent.duration._
+
+class OkHttpMonixWebSocketTest extends WebSocketTest[Task] with WebSocketStreamingTest[Task, MonixStreams] {
+  override val streams: MonixStreams = MonixStreams
+  override implicit val backend: SttpBackend[Task, MonixStreams with WebSockets] =
+    OkHttpMonixBackend().runSyncUnsafe()
+  override implicit val convertToFuture: ConvertToFuture[Task] = convertMonixTaskToFuture
+  override implicit val monad: MonadError[Task] = TaskMonadAsyncError
+
+  override def throwsWhenNotAWebSocket: Boolean = true
+
+  it should "error if incoming messages overflow the buffer" in {
```

Comment:
this is also in `AsyncHttpClientWebSocketTest`, maybe move to a trait? (`WebSocketBufferOverflowTest`)